slack/client.go

```diff
@@ -415,6 +416,59 @@ func (c *slackClient) loadFromStorage() error {
 	return nil
 }
 
+// Broadcast sends message, and returns the number of
+// processed bots, conversations, errors and error occurred
+func (c *slackClient) Broadcast(msg flamingo.Sendable, isValid flamingo.CanBeBroadcasted) (
+	clientBotsCount uint64, conversationsCount uint64, errorsFound uint64, err error,
+) {
+	var convs, errors uint64
+	for _, clientBot := range c.bots {
+		convs, errors = broadCastTo(clientBot.(*botClient), msg, isValid)
+		errorsFound += errors
+		conversationsCount += convs
+		if convs > 0 {
+			clientBotsCount++
+		}
+	}
+
+	if errorsFound > 0 {
+		err = fmt.Errorf("%d messages could not be delivered", errorsFound)
+	}
+
+	return
+}
+
+func broadCastTo(botClient *botClient, msg flamingo.Sendable, isValid flamingo.CanBeBroadcasted) (
+	sentCount uint64, errorsFound uint64,
+) {
+	for _, conversation := range botClient.conversations {
+		if isValid(botClient.id, conversation.channel, msg) {
+			err := send(conversation.createBot(), msg)
+			sentCount++
+			if err != nil {
+				errorsFound++
```

Comment:
it is counted the processed channels.
The goal is to provide this info at the end of the process:
Teams / channels / ocurred errors